Assets/Vimeo/Scripts/Recorder/VimeoRecorder.cs

```diff
@@ -62,8 +66,10 @@ public void CancelRecording()
             Destroy(publisher);
         }
 
-        private void PublishVideo(string filePath)
+        //Used if you want to publish the latest recorded video
+        public void PublishVideo()
         {
+            string filePath = encoder.GetVideoFilePath();
```

Comment:
minor, but you could just skip this string assignment and put it into the code directly. e.g.

```c#
 publisher.PublishVideo(encoder.GetVideoFilePath());
```